community-operators/prometheus/prometheusoperator.0.27.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,288 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: prometheusoperator.0.27.0
+  namespace: placeholder
+  annotations:
```

Comment:
Additional UI required annotations can be found [here](https://github.com/operator-framework/community-operators/blob/master/docs/marketplace-required-csv-annotations.md).

Reply:
@SamiSousa https://github.com/operator-framework/community-operators/blob/master/docs/marketplace-required-csv-annotations.md shows 404, do you mind to reference the correct link?

Reply:
Sorry, here's the link: `https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md`